blocks/library/image/editor.scss

```diff
@@ -29,6 +29,7 @@
 
 	.wp-block-image.is-focused & {
 		display: block;
+		z-index: 1;
```

Comment:
Could we use a variable in the shared `editor/assets/stylesheets/_z-index.scss`? (see other applied z-index)

Reply:
Also maybe we want to apply this to the resize handles only to keep the "sibling inserter" above the image

Reply:
Thanks, @youknowriad!  I've updated this PR to include the `z-index` variables. 

Right now the `z-index` only applies to the handlers and not the image.  It's being applied to:

>.wp-block-image.is-focused .wp-block-image__resize-handler-bottom-left, .
wp-block-image.is-focused .wp-block-image__resize-handler-bottom-right, 
.wp-block-image.is-focused .wp-block-image__resize-handler-top-left, 
.wp-block-image.is-focused .wp-block-image__resize-handler-top-right

Is there something else you were thinking of?